src/modules/addReserve.js

```diff
@@ -0,0 +1,18 @@
+const addReserve = (dataid, name, sDate, eDate) => {
+  fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/cXyK0mK0Ha1BkQqOi8Kz/reservations',
+    {
+      method: 'post',
+      body: JSON.stringify({
+
+        item_id: dataid,
+        username: name,
+        date_start: sDate,
+        date_end: eDate,
+
+      }),
+      headers: { 'content-type': 'application/json' },
+    })
+    .then((resc) => resc.text());
+};
+
+export default addReserve;
```

Comment:
- [ ] Please kindly implement **async** and **await** with the fetch method to send and receive data from the API. The **async** and **await** keywords enable asynchronous, promise-based behavior to be written in a cleaner style. ℹ️ 

Kindly implement this for **addComment.js**, **fetchApiRes.js** and **fetchapi.js** modules as well (you may want to refactor your code easily as we have it in **addlikes.js**).

## Project requirement
![DSC_7491](https://user-images.githubusercontent.com/103386183/218141584-93002965-ef99-43eb-a162-1e6373612e6c.JPG)